Sofa/Component/Constraint/Lagrangian/Solver/src/sofa/component/constraint/lagrangian/solver/GenericConstraintSolver.cpp

```diff
@@ -393,7 +393,7 @@ void GenericConstraintSolver::buildSystem_matrixAssembly(const core::ConstraintP
     msg_info() << " computeCompliance_done "  ;
 }
 
-void GenericConstraintSolver::rebuildSystem(SReal massFactor, SReal forceFactor)
```

Comment:
Yes ! 

It was a mistake to pass it to SReal because this method is overriding one with the following signature:  ConstraintSolver::rebuildSystem(double, double).